packages/twitter/twitter_client.js

```diff
@@ -38,9 +36,12 @@ Twitter.requestCredential = function (options, credentialRequestCompleteCallback
     }
   }
 
-  // Handle force login (request the user to enter their credentials)
-  if (options && options.force_login) {
-    loginPath += "&force_login=true";
+  // Support additional, permitted parameters
+  var addlParams = Twitter.validParamsAuthenticate;
+  for (var i = 0, len = addlParams.length; i < len; i++) {
+    if (options && options[addlParams[i]]) {
+      loginPath += "&" + addlParams[i] + "=" + encodeURIComponent(options[addlParams[i]]);
+    }
```

Comment:
`Twitter.validParamsAuthenticate.forEach(function (param) { ... })` would be fine here, I think.